dbclient/TableACLsClient.py

```diff
@@ -231,8 +231,10 @@ def export_table_acls(self, db_name='', table_alcs_dir='table_acls/'):
 
         user_name = self.get_current_username(must_be_admin=True)
         if self.DB_ADMIN_SUFFIX in user_name:
-            user_name = "admin"
-        export_table_acls_workspace_path = f"/Users/{user_name}/tmp/migrate/Export_Table_ACLs.py"
+            notebook_parent_path = ""
+        else:
+            notebook_parent_path = f"/Users/{user_name}"
+        export_table_acls_workspace_path = f"{notebook_parent_path}/tmp/migrate/Export_Table_ACLs.py"
```

Comment:
why is this change showing up here ?

Reply:
oh.. I am not sure. hopefully merging this will resolve it